package.json

```diff
@@ -10,11 +13,13 @@
     "preinstall": "cd generator-theia && npm install",
     "prepare": "npm install generator-theia && npm run bootstrap && npm run clean && npm run build",
     "prepublishOnly": "npm run test",
-    "bootstrap": "node scripts/generate.js && lerna bootstrap --hoist",
+    "bootstrap": "node scripts/generate.js && lerna bootstrap --hoist -- --no-optional",
```

Comment:
Bunyan has some optional dependencies for dtracing and so on, we don't use it, by passing `--no-optional` we avoid downloading unnecessary dependencies.